$OpenBSD: patch-plugins_rdp_rdp_plugin_c,v 1.2 2020/03/28 07:24:37 robert Exp $

Revert https://github.com/FreeRDP/Remmina/commit/9df60caeb4f62303c29d69021991e3de14802ba1#diff-7ea33c789952583adb2e74168c346a6a
FreeRDP_GatewayAccessToken was introduced in FreeRDP rc2

Partly revert https://github.com/FreeRDP/Remmina/commit/1ce9d3e7ae99baadc5bb810061cc177460eec992
AllowUnanouncedOrdersFromServer is not available in our FreeRDP yet

Index: plugins/rdp/rdp_plugin.c
--- plugins/rdp/rdp_plugin.c.orig
+++ plugins/rdp/rdp_plugin.c
@@ -926,17 +926,12 @@ static gboolean remmina_rdp_main(RemminaProtocolWidget
 	if (rfi->settings->GatewayEnabled)
 		freerdp_set_gateway_usage_method(rfi->settings,
 			remmina_plugin_service->file_get_int(remminafile, "gateway_usage", FALSE) ? TSC_PROXY_MODE_DETECT : TSC_PROXY_MODE_DIRECT);
-
-	freerdp_set_param_string(rfi->settings, FreeRDP_GatewayAccessToken,
-		remmina_plugin_service->file_get_string(remminafile, "gatewayaccesstoken"));
-
-	rfi->settings->AuthenticationLevel = remmina_plugin_service->file_get_int(
-		remminafile, "authentication level", rfi->settings->AuthenticationLevel);
-
 	/* Certificate ignore */
 	rfi->settings->IgnoreCertificate = remmina_plugin_service->file_get_int(remminafile, "cert_ignore", 0);
 
+#ifdef notyet
 	rfi->settings->AllowUnanouncedOrdersFromServer = remmina_plugin_service->file_get_int(remminafile, "relax-order-checks", 0);
+#endif
 
 	rfi->settings->GlyphSupportLevel = ( remmina_plugin_service->file_get_int(remminafile, "glyph-cache", 0) ? GLYPH_SUPPORT_FULL : GLYPH_SUPPORT_NONE );
 
@@ -1713,7 +1708,9 @@ static const RemminaProtocolSetting remmina_rdp_advanc
 	{ REMMINA_PROTOCOL_SETTING_TYPE_CHECK,	    "enableproxy",	       N_("Enable proxy support"),		TRUE,	NULL,		NULL},
 #endif
 	{ REMMINA_PROTOCOL_SETTING_TYPE_CHECK,	    "disableautoreconnect",    N_("Disable automatic reconnection"),	TRUE,	NULL,	        NULL},
+#ifdef notyet
 	{ REMMINA_PROTOCOL_SETTING_TYPE_CHECK,	    "relax-order-checks",      N_("Relax Order Checks"),	        TRUE,	NULL,	        NULL},
+#endif
 	{ REMMINA_PROTOCOL_SETTING_TYPE_CHECK,	    "glyph-cache",             N_("Glyph Cache"),	                TRUE,	NULL,	        NULL},
 	{ REMMINA_PROTOCOL_SETTING_TYPE_END,	    NULL,			NULL,					FALSE,	NULL,		NULL}
 };
